### ISSUE TITLE
Create website and update the about box URLs

### ISSUE THREAD

**Issue by byuu** [higan-emu/higan#3]
It's very important to put together an official website for higan and byuu, with features, screenshots, and download links. It could also prove helpful to include documentation for each emulator on said page at some point.

Both higan and byuu have a URL in the about box which points to the higan-emu GitHub repository. They should be updated to point at the new webpages once this is done.

**Comment by Screwtapello** [comment]
I notice https://higan-emu.com/ is now a thing, backed by this repo: https://github.com/higan-emu/higan-emu.github.io

@MerryMage, what are your plans here? Compared to byuu's check-list, we're still missing screenshots, documentation, and updating higan/byuu to point at the correct website. Should we work through those in this issue? Would you prefer to create issues for "add screenshots" and "think about documentation" to the website repo, and leave this issue just for updating the URLs in the code?

**Comment by MerryMage** [comment]
We could work through those in this issue.

**Comment by Screwtapello** [comment]
### Screenshots

The higan `README.md` links to a bunch of screenshots hosted on byuu.org; it shouldn't be difficult to copy them into the website repo and have them show up on the homepage somewhere. Would you like to do that yourself, or are you happy to let me hack around and get something working?

### Documentation

Currently the website links to the documentation I wrote at https://higan.readthedocs.io/ which is out-of-date, but still has some value. My strawman proposal for the documentation is:

- create a new mkdocs project in the higan repository, like it was before, so the documentation can be versionned and tagged alongside the code it documents
- keep hosting documentation at ReadTheDocs, because there's already links to it, and because the "switch to previous versions of the documentation" is a nice feature it would be hard for us to replicate
- make the website link directly to the docs, rather than going through a redirect

I'm open to changing all of those decisions, if other maintainers would prefer something different. Docs in a separate repo? Hosted on the higan site, instead of a third-party server? Use Sphinx or GitHub Wiki or mdbook instead of mkdocs? Let me know! Or if you're happy with my proposal above, let me know that too.

### About dialog updates

This should be pretty simple to do. Technically it can be done at any time, it doesn't require the above tasks, but it'd be a nice way to cap off the other work.

**Comment by MerryMage** [comment]
> Would you like to do that yourself, or are you happy to let me hack around and get something working?

I do not mind anyone stepping in if they wish to, especially given that thanks to *gestures at the current situation* I am busier than usual.

**Comment by Screwtapello** [comment]
Since byuu's actual, real retirement, we now have control over https://byuu.org/higan. The higan-ui, byuu-ui and icarus about boxes all point at https://higan.dev, which we do *not* have direct control over, but which redirects to https://byuu.org/higan.

We still need to write documentation and think about where to host it, but that can be a separate issue.
